crates/ark/src/reticulate.rs

```diff
@@ -18,21 +19,27 @@ use uuid::Uuid;
 
 use crate::interface::RMain;
 
-static RETICULATE_COMM_ID: LazyLock<Mutex<Option<String>>> = LazyLock::new(|| Mutex::new(None));
+static RETICULATE_SOCKET: LazyLock<Mutex<Option<CommSocket>>> = LazyLock::new(|| Mutex::new(None));
```

Comment:
I'd have made that as minimal as possible so that only the service is able to handle messages but I guess we can also just be careful.

Reply:
Ok, I can probably just keep the channel the `outbound.tx` channel here.